### DISCUSSION TITLE
How to get model update time?

### DISCUSSION THREAD

**Discussion by daanaerts** [open-meteo/open-meteo#910] [Q&A] [answered]
Hey, weather models run at different intervals, e.g. every 6 hours. 

If I understand correctly, most, if not all, agencies delay publishing new runs by an hour or so to their open access apis. 

Is there a way to see the issue timestamp for a given value? e.g. I query ecmwf_ifs025 temperature_2m forecast for today, which model run time is my value obtained from?

**Comment by patrick-zippenfenig** [comment, marked as answer]
Hi,

model update times are not yet exposed. This is something I would like to add in the future, but due to the current design of Open-Meteo it is not as easy as it sounds. As a workaround you can check the updates times on the open-data distribution. https://github.com/open-meteo/open-data

Open-Meteo is optimised to download and process data as soon as possible. Typically is only takes 10 minutes to process and copy all data.

Actually, most open-data weather distributions are published as soon as possible. Only ECMWF has a delay of ~1 hour.
